config/flipt.schema.json

```
@@ -637,6 +634,11 @@
           "type": "boolean",
           "default": true,
           "deprecated": true
+        },
+        "default_theme": {
+          "type": "string",
```

Similar suggestion here to the CUE one:
```suggestion
          "type": "string",
          "enum": ["light", "dark", "system"],
```